packages/app-mobile/components/NoteEditor/CodeMirrorDecorator.ts

```diff
@@ -0,0 +1,161 @@
+/**
```

Comment:
camelCase - "codeMirrorDecorators.ts" (and should be plural if there's more than one decorator)